src/components/organisms/AssetActions/Pool/index.tsx

```diff
@@ -196,17 +204,19 @@ export default function Pool({ ddo }: { ddo: DDO }): ReactElement {
     }
     init()
 
-    // Re-fetch price periodically, triggering re-calculation of everything
-    const interval = setInterval(() => refreshPrice(), refreshInterval)
-    return () => clearInterval(interval)
+    // TODO why is this again here ?!?!?!
```

Comment:
use case was so that people could stay with opened Pool widget and everything in there updates in the shown interval. So all subcomponents react to the price change, and then recalculate

Reply:
removed